### PR TITLE
Update Docs

### LINKED ISSUE #1326: Remove/Update stale documentation
We are not maintaining helm charts under [this](https://docs.kanister.io/helm.html#kanister-enabled-applications) section, we are using upstream helm charts to install the  applications. We should update these docs to point to examples in kanister

### PR DESCRIPTION
**Pull request by ihcsim**
## Change Overview

This PR introduces the following documentation updates:

1. Simplify installation steps
2. Remove 'usage' section and promote its subsections to main sections
3. Add 'Next Step' to tutorial to point to example Helm charts on GitHub
4. Remove obsolete Helm instructions page
5. Remove redundant prefix in page titles
6. Update copyright info. in footer

Signed-off-by: Ivan Sim <ivan.sim@kasten.io>

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [x] :world_map: Documentation
- [ ] :robot: Test

## Issues

Fix #1326 

## Test Plan

Not required - only doc changes.

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -1,91 +1,82 @@
...
-Deploy a Released Version
--------------------------
+  helm -n kanister upgade --install kanister --create-namespace kanister/kanister-operator \
+    --set controller.updateCRDs=false
 
-To deploy a released version of the controller, issue the command below. Modify
-the namespace fields in ``bundle.yaml.in`` to deploy in a namespace of your
-choice. By default, the controller will be deployed into the ``default``
-namespace.
+This option let Helm manage the CRD resources.
```
Comment (PrasadG193) [inline docs/install.rst:75]:
```suggestion
This option lets Helm manage the CRD resources.
```
[nit]

File: docs/install.rst [hunk 2]
```diff
@@ -1,91 +1,82 @@
...
 
-.. substitution-code-block:: bash
+Building and Deploying from Source
+==================================
 
-   # Deploy controller version |version| to Kubernetes
-   $ make deploy VERSION="|version|"
+Follow the instructions in the ``BUILD.md`` file in the
+`Kanister GitHub repository <https://github.com/kanisterio/kanister/blob/master/BUILD.md>`_
+to build Kanister from source.
```
Comment (PrasadG193) [inline docs/install.rst:82]:
```suggestion
to build Kanister from the source.
```
[nit]

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+## Kanister Documentation
+
+This folder contains the source of the Kanister documentation at
+https://docs.kanister.io
+
+## Building The Source
+
+The content are written in the
+[reStructuredText](https://docutils.sourceforge.io/rst.html) format.
```
Comment (pavannd1) [inline docs/README.md:9]:
```suggestion
The content is written in the
[reStructuredText](https://docutils.sourceforge.io/rst.html) format.
```
[nit]

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+```sh
+make docs
+```
+
+The `docs` target uses the `ghcr.io/kanisterio/docker-sphinx` public image to
+generate the HTML documents and store them in your local `/docs/_build/html`
+folder.
+
+## Language, Grammar and Tone
```
Comment (pavannd1) [inline docs/README.md:21]:
```suggestion
## Language, Grammar, and Tone
```
[nit] optional - feel free to ignore

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+When updating the documentation content, please follow these guidelines on the
+language, grammar and tone style:
```
Comment (pavannd1) [inline docs/README.md:24]:
```suggestion
language, grammar, and tone style:
```
[nit] feel free to ignore

File: docs/index.rst [hunk 6]
```diff
@@ -11,19 +11,19 @@ Kanister: Application-Specific Data Management
...
    :target: https://github.com/kanisterio/kanister/actions
 
-Kanister allows domain experts to capture application-specific data management
-tasks in Blueprints which can be easily shared and extended. The framework takes
-care of the tedious details around execution on Kubernetes and presents a
-homogeneous operational experience across applications at scale.
+Kanister is a data protection workflows management tool. It provides a set of
+cohesive APIs for defining and curating data operations by abstracting away
+tedious details around executing data operations on Kubernetes. It's extensible
+and easy to install, operate and scale.
```
Comment (pavannd1) [inline docs/index.rst:17]:
```suggestion
Kanister is a data protection workflow management tool. It provides a set of
cohesive APIs for defining and curating data operations by abstracting away
tedious details around executing data operations on Kubernetes. It's extensible
and easy to install, operate and scale.
```

File: docs/install.rst [hunk 7]
```diff
@@ -1,91 +1,82 @@
...
-   $ kanctl create profile s3compliant --bucket <bucket> --access-key ${AWS_ACCESS_KEY_ID} \
-                                       --secret-key ${AWS_SECRET_ACCESS_KEY}               \
-                                       --region <region>                                   \
-                                       --namespace kanister
+.. note::
+  Kanister is guaranteed to work with the 3 most-recent versions of Kubernetes.
+  For example, if the latest version of Kubernetes is 1.24, Kanister will work
+  with 1.24, 1.23 and 1.22. Support for older versions are determined on a
+  best-effort basis. If you are using an older version of Kubernetes, please
+  consider upgrading to a newer version.
```
Comment (pavannd1) [inline docs/install.rst:44]:
```suggestion
  Kanister is guaranteed to work with the 3 most recent versions of Kubernetes.
  For example, if the latest version of Kubernetes is 1.24, Kanister will work
  with 1.24, 1.23, and 1.22. Support for older versions is determined on a
  best-effort basis. If you are using an older version of Kubernetes, please
  consider upgrading to a newer version.
```

File: docs/install.rst [hunk 8]
```diff
@@ -1,91 +1,82 @@
...
-   $ make release-controller REGISTRY=<MY REGISTRY>
+Managing Custom Resource Definitions (CRDs)
+===========================================
 
-   # Deploy the controller to your Kubernetes repo
-   $ make deploy REGISTRY=<MY REGISTRY>
+The default RBAC settings in the Helm chart permits Kanister to manage and
+auto-update its own custom resource definitions, to ease user's operation
+burden. If your setup requires the removal of these settings, you will have to
+install Kanister with the ``--set controller.updateCRDs=false`` option:
```
Comment (pavannd1) [inline docs/install.rst:68]:
```suggestion
The default RBAC settings in the Helm chart permit Kanister to manage and
auto-update its own custom resource definitions, to ease the user's operation
burden. If your setup requires the removal of these settings, you will have to
install Kanister with the ``--set controller.updateCRDs=false`` option:
```

File: docs/tutorial.rst [hunk 9]
```diff
@@ -623,3 +623,15 @@ and manually updating the action set to use artifacts created by the
...
+=========
+
+Congratulation! You have reached the end of this long tutorial! 🎉🎉🥳🥳
+
+Don't stop here. There are many more example blueprints on the Kanister GitHub
+`repository <https://github.com/kanisterio/kanister/tree/master/examples>`_
+to explore. Use them to help you define your next blueprint.
+
+You would love to hear from you. If you have any feedback or questions, find us
+on Slack at `kanisterio.slack.com <https://kanisterio.slack.com>`_.
```
Comment (pavannd1) [inline docs/tutorial.rst:637]:
```suggestion
Congratulations! You have reached the end of this long tutorial! 🎉🎉🥳🥳

Don't stop here. There are many more example blueprints on the Kanister GitHub
`repository <https://github.com/kanisterio/kanister/tree/master/examples>`_
to explore. Use them to help you define your next blueprint.

We would love to hear from you. If you have any feedback or questions, find us
on Slack at `kanisterio.slack.com <https://kanisterio.slack.com>`_.
```

**Review by pavannd1 — COMMENTED** [review]
Some nits

File: docs/README.md (hunk 3 above)
Comment (pavannd1) [reply docs/README.md:9]:
```suggestion
The contents are written in the
[reStructuredText](https://docutils.sourceforge.io/rst.html) format.
```
@ihcsim We should probably do one of these changes.

Comment (ihcsim) [reply docs/README.md:9]: ah sorry, i missed that one. Will fix.

**Review by pavannd1 — APPROVED** [review]
One minor suggestion but looks good otherwise 👍🏼

**Pull request merged**